easybuild/easyconfigs/p/PyExtensions/PyExtensions-2.7.15.6-CrayGNU-19.10.eb

```
@@ -15,42 +11,33 @@ toolchain = {'name': 'CrayGNU', 'version': '19.10'}
 toolchainopts = {'pic': True, 'verbose' : False }
 
 dependencies = [
-    ('cray-python/%s' % version, EXTERNAL_MODULE),
+    ('cray-python/2.7.15.6', EXTERNAL_MODULE),
```

```suggestion
    ('cray-python/2.7.15.7', EXTERNAL_MODULE),
```
shouldn't we update to 2.7.15.7 instead?

---

No, because we have two `PyExtensions` recipes for python2. One for 2.7.15.6 and one for 2.7.15.7. And both are used in the production list